neo4j/CHANGELOG.md

```diff
@@ -1,2 +1,3 @@
-# CHANGELOG - Neo4j2
+# CHANGELOG - neo4j
 
+21 July 2021 - Updated Neo4j integration to support 4.x
```

Comment:
We should not be trying to match the integration version with the product version.

The accurate next version should be 2.0.0 since we are adding feature support. We shouldn't be removing the previous changelog line 1.0.1, or else we lose historical context.

The changelog should look like:

```
## 2.0.0 / 2021-07-21
* [Added] Updated Neo4j integration to support 4.x. See [#944](https://github.com/DataDog/integrations-extras/pull/944)

## 1.0.1 / 2021-07-28

* [Added] Update neo4j check to use requests wrapper. See [#894](https://github.com/DataDog/integrations-extras/pull/894).
```

We should maybe detail some more of the breaking changes introduced in this PR in the changelog for reference.